src/containers/ContactList/index.tsx

```diff
@@ -110,19 +112,24 @@ export class Component extends React.Component<ComponentProps, {}> {
         this.props.tree.isPending.set(true);
 
         try {
-            try {
-                await CallService.requestPermissions(false);
-            } catch (err) {
-                return Navigation.showOverlay({
-                    component: {
-                        name: 'td.Modal',
-                        passProps: { text: err.message },
-                    },
-                });
+            const callResponse = await CallService.makeOutgoingCall(
```

Comment:
Don't use try here. Errors are handled in `makeOutgoingCall` service. Here you have a RemoteDataResult which can be a success or a failure. 

Just show an error in case of a failure.
And in all cases restore `isPending` flag.